### ISSUE TITLE
Regarding softmax layer

### ISSUE THREAD

**Issue by JackChen890311** [NaJaeMin92/pytorch_DANN#7]
The softmax layer makes models hard to train. If you faced the same problem try to remove the last softmax layer in Classifier and Discriminator.

**Comment by NaJaeMin92** [comment]
Hi, @JackChen890311 

Thank you for pointing that out. 

Since `LogSoftmax` is already included in `nn.CrossEntropyLoss`, it is redundant. 

Please check the updated code.

**Comment by JackChen890311** [comment]
Thanks for the update and the explanation!
